app/src/main/java/edu/uc/clemenks/keepingupwithschool/dto/LocationDetails.kt

```diff
@@ -1,4 +1,4 @@
 package edu.uc.clemenks.keepingupwithschool.dto
 
-data class LocationDetails (val longitude: String, val latitude: String) {
+data class LocationDetails (var longitude: String, var latitude: String) {
```

Comment:
Keep these as val, per the Kotlin Coding Conventions: 
https://kotlinlang.org/docs/reference/coding-conventions.html#immutability
"Immutability
Prefer using immutable data to mutable. Always declare local variables and properties as val rather than var if they are not modified after initialization."